Assets/Scripts/UIManager.cs

```diff
@@ -0,0 +1,139 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+using UnityEngine.UI;
+
+
+public class UIManager : MonoBehaviour
+{
+    // Start is called before the first frame update
+
+
+ 
+    public TextMeshProUGUI playerOneScoreLabel;
```

Comment:
1. All the variables are not matching our naming scheme, please read the c# style guide on the repo's root
2. From the basic principle of encapsulation you should never make variables public unless for special use cases. If you want to access the variable in the unity editor u can add a `[Serializefield]` attribute to the variable, please take reference to other's script.